config-templates/google/gcp-compute-persistent-disk-csi-driver/1.8.0/deployment.yaml

```diff
@@ -0,0 +1,848 @@
+              failureThreshold: 1
+              httpGet:
+                path: /healthz/leader-election
+                port: http-endpoint
+              initialDelaySeconds: 10
+              periodSeconds: 20
+              timeoutSeconds: 10
+            name: csi-resizer
+            ports:
+            - containerPort: 22013
+              name: http-endpoint
+              protocol: TCP
+            volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+          - args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --metrics-address=:22014
+            - --leader-election
+            - --leader-election-namespace=$(PDCSI_NAMESPACE)
+            - --timeout=300s
+            env:
+            - name: PDCSI_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.1
+            name: csi-snapshotter
+            volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+          - args:
+            - --v=5
+            - --endpoint=unix:/csi/csi.sock
+            env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/cloud-sa/cloud-sa.json
+            image: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:v1.7.2
+            name: gce-pd-driver
+            volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+            - mountPath: /etc/cloud-sa
+              name: cloud-sa-volume
+              readOnly: true
+          hostNetwork: true
+          nodeSelector:
+            kubernetes.io/os: linux
+          priorityClassName: csi-gce-pd-controller
+          serviceAccountName: csi-gce-pd-controller-sa
+          volumes:
+          - emptyDir: {}
+            name: socket-dir
+          - name: cloud-sa-volume
+            secret:
+              secretName: cloud-sa
+  - apiVersion: apps/v1
+    kind: DaemonSet
+    metadata:
+      name: csi-gce-pd-node
+      namespace: gce-pd-csi-driver
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.8.0
+    spec:
+      selector:
+        matchLabels:
+          app: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/version: 1.8.0
+      template:
+        metadata:
+          labels:
+            app: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/version: 1.8.0
+          annotations:
+            autoUpdate: "AUTO-UPDATE-ENABLED"
+        spec:
+          containers:
+          - args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
```

Comment:
I think here also we need to change the path to /var/data

Reply:
Changed here: https://github.com/IBM/ibm-satellite-storage/pull/221/commits/0f41c691643168e11046fa58c0916fd334695a3d